app/src/main/java/com/kamron/pogoiv/Pokefly.java

```diff
@@ -975,14 +944,6 @@ private void setTextColorbyPercentage(TextView text, int value) {
         }
```

Comment:
Yes we want it in one place. And I'm betting we wanted out of Pokefly. But I decided to wait for the approach we are going to take for Pokefly rework.
Maybe the best way to deal with this case in particular is to extract it into a helper class?